docs/resources/service-providers.md

```diff
@@ -88,156 +201,97 @@ You will be asked to create a password (at least 8 characters) for this key-pair
 - `TYPE`: Type of your key, always `local`. 
 - `ADDRESS`: Your address. Used to receive funds.
 - `PUBKEY`: Your public key. Useful for validators.
-- `MNEMONIC`: 24-words phrase. **Save this mnemonic somewhere safe**. It is used to recover your private key in case you forget the password.
+- `MNEMONIC`: 24-words phrase. **Save this mnemonic somewhere safe**. It is used to recover your private key in case you forget the password. It is displayed at the end of the output.
 
-You can see all your available keys by typing:
+You can see all available keys by typing:
 
 ```bash
 gaiad keys list
 ```
 
-#### Checking your balance
-
-After receiving tokens to your address, you can view your account's balance by typing:
+If you want to add a key to your keyring that imports a mnemonic, use the `--recover` flag.
 
 ```bash
-gaiad query account <YOUR_ADDRESS>
+gaiad keys add <your_key_name> --recover
 ```
 
-*Note: When you query an account balance with zero tokens, you will get this error: No account with address <YOUR_ADDRESS> was found in the state. This is expected! We're working on improving our error messages.*
+#### Check your Account
 
-#### Sending coins via the CLI
-
-Here is the command to send coins via the CLI:
+You can view your account by using the `query account` command.
 
 ```bash
-gaiad tx send <from_key_or_address> <to_address> <amount> \
-    --chain-id=<name_of_testnet_chain> 
+gaiad query account <YOUR_ADDRESS>
 ```
 
-Parameters:
-
-- `<from_key_or_address>`: Key name or address of sending account.
-- `<to_address>`: Address of the recipient.
-- `<amount>`: This parameter accepts the format `<value|coinName>`, such as `10faucetToken`.
-
-Flags:
-
-- `--chain-id`: This flag allows you to specify the id of the chain. There will be different ids for different testnet chains and main chain.
-
-#### Help
-
-If you need to do something else, the best command you can run is:
+It will display your account type, account number, public key and current account sequence.
```

Comment:
```suggestion
Your account type displays with your account number, public key, and current account sequence.
```

Reply:
It displays
- account type
- account number
- public key
- current account

with the new wording it sounds like the account type contains the other data.